### ISSUE TITLE
Оформление отступов в коде

### ISSUE THREAD

**Issue by Artemka091102** [pascalabcnet/pascalabcnetide#279]
Независимо от выставленного параметра "Сервис-Настойки-Редактор-Конвертировать Tab в пробелы" при переносе строки после ключевых слов var, begin и прочих IDE генерирует отступ в 2 пробела вместо выставленного в параметре их количества

**Comment by miks1965** [comment]
Отступы всегда одни - два пробела. 
А конвертация табов в пробелы - да, указывается сколько пробелов вместо табов.

Это разные вещи.

**Comment by SunSerega** [comment]
> Это разные вещи.

Разные с чем? Эта настройка работает и при нажатии кнопки Tab вручную. А issue о том что авто-отступы (которые должны работать как автоматическое нажатие Tab) эту настройку игнорируют и ставят не такой же отступ как во всём остальном документе.

**Comment by miks1965** [comment]
Авто-отступы не должны работать как автоматическое нажатие Tab - это вы придумали. Мы лишь конвертируем старые tab в файлах в пробелы. А отступы всегда должны быть равны 2

**Comment by SunSerega** [comment]
Если вы только старые tab символы конвертируете и если "отступы всегда должны быть равны 2" - новое нажатие кнопки Tab должно добавлять только 2 пробела. А добавляет столько - сколько в настройке.

**Comment by miks1965** [comment]
Аа, раз так, то наверное. Но это всё равно плохая идея. Наши школьники перестраивают IDE из хулиганских соображений. И для следующих автоотступ будет 500. А так он всегда 2

**Comment by SunSerega** [comment]
У вас такие настройки в 1 файле хранятся. Может возможно этот файл настроек автоматически сбрасывать? В конце концов, сделать .bat файл с 1 строчкой внутри и засунуть в папку автозапуска...

**Comment by miks1965** [comment]
Легче оставить как есть чем огород городить

**Comment by SunSerega** [comment]
Да, всегда легче оставить чем пытаться что то сделать. Но тем не менее сейчас неоднородность в текущем функционале. Ну, пусть уже хоть issue висит...

**Comment by samuraiGH** [comment]
> @miks1965 Наши школьники перестраивают IDE из хулиганских соображений. И для следующих автоотступ будет 500

По такому же принципу надо производство ножей прекратить. А то из хулиганских  побуждений ими людей режут
